### DISCUSSION TITLE
Basic filtering support

### DISCUSSION THREAD

**Discussion by xmo-odoo** [CtrlSpice/otel-desktop-viewer#156] [Ideas]
Yeha, thanks for the desktop viewer, it's a really convenient way to get basic traces display (especially when just getting started).

There's one thing I've been wondering: when doing tests which generate a lot of traces, or when auto-instrumentation generates tons of spans out of the box the viewer can get pretty unwieldy, would you be in favor of adding basic filtering (e.g. fayt of the traces / spans, ability to zoom in on a span inside a trace) or do you consider that to be out of scope for the desktop viewer, and a sign that the user needs to "graduate" to a more complete solution?

**Comment by CtrlSpice** [comment]
Hey! So, yes absolutely. 

For some context, this started out as very much a learner project for me, but now that I have the skills (and a bit of time) to do so, I started working on revamping it to something more usable and complete! :) Filtering is definitely on my radar, as well as data persistence, logs and metrics, etc.
